keyboards/18bluetoothpad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,58 @@
+/* Keymap _0: (Base Layer) Default Layer
+   * .-----------.
+   * |PGUP | PGDN| 
+   * |-----------------------.
+   * |  7  |  8  |  9  | MO1 |
+   * |-----|-----|-----|-----|
+   * |  4  |  5  |  6  |  +  |
+   * |-----|-----|-----|-----|
+   * |  1  |  2  |  3  |  —  |
+   * |-----|-----|-----|-----|
+   * |  0  |  .  | DEL |Enter|
+   * .-----------------------.
+   */
+     /* Keymap _1: (Second Layer) second Layer
+   * .---------------.
+   * |NUMLOCK|Calc.  | 
+   * |--------------------------------.
+   * |RGB_TOG|RGB_MOD|RGB_M_K|RGB_M_X |
+   * |-------|-------|-------|--------|
+   * |RGB_SAI|RGB_SAD|RGB_HUI|RGB_HUD |
+   * |-------|-------|-------|--------|
+   * |RGB_VAI|RGB_VAD|RGB_SPI|RGB_SPD |
+   * |-------|-------|-------|--------|
+   * |RGB_SPI|RGB_SPD|RGB_TOG|KC_TRNS |
+   * .--------------------------------.
+*/
+
+#include QMK_KEYBOARD_H
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
+        KC_KP_7, KC_KP_8, KC_KP_9, MO(1),
+        KC_P4, KC_P5, KC_P6, KC_KP_PLUS,
+        KC_P1, KC_P2, KC_P3, KC_KP_MINUS,
+        KC_P0, KC_PDOT,KC_DELETE, KC_KP_ENTER,
+        KC_PGUP, KC_PGDOWN),
+    [1] = LAYOUT_all(
+        RGB_TOG, RGB_MOD, RGB_M_K, RGB_M_X,
+        RGB_SAI, RGB_SAD, RGB_HUI, RGB_HUD,
+        RGB_VAI, RGB_VAD, RGB_SPI, RGB_SPD,
+        RGB_SPI, RGB_SPD, RGB_TOG, KC_TRNS,
+        KC_NUMLOCK, KC_CALCULATOR),
```

Comment:
You need two more layers here (filled with `KC_TRNS`) for the default `DYNAMIC_KEYMAP_LAYER_COUNT` of 4.

Reply:
immediately，sir